packages/design-system-web/src/index.html

```diff
@@ -12,6 +12,7 @@
 </head>
 
 <body mode="ios">
+  Hello
```

Comment:
Hi!
... why?

Reply:
The whole file is not needed anymore, now that the reload time with storybook is acceptable.
I delete it as well as the command `npm run dev:stencil-only`